packages/@aws-cdk/aws-sns/lib/subscriber.ts

```diff
@@ -1,9 +1,19 @@
-import { Construct } from '@aws-cdk/cdk';
+import { SubscriptionOptions } from './subscription';
 import { ITopic } from './topic-base';
 
+/**
+ * Subscription configuration
+ */
+export interface TopicSubscriptionConfig extends SubscriptionOptions {
+  /**
+   * The id of the construct that is being subscribed to the topic.
+   */
+  readonly id: string;
```

Comment:
should this be called `subscriberId` and the help text should explain what is being done with it